data/mil2525d/utilities/style-utilities/style-file-utilities/CreateAllMilitaryStyles.bat

```diff
@@ -1,57 +0,0 @@
-:: !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
@csmoore, I guess the answer to this depends on what we have decided regarding situations like this.  Have we established a consistent approach in past situations as to deciding which file is the managed version, one in a deployment zip or install versus one not,  Perhaps we should manage both and strive to keep them identical.